templates/environments/test-environment-template.js

```diff
@@ -1,14 +1,15 @@
-function makeTestEnvironment(_, simple) {
+function makeTestEnvironment(_, simpleMock) {
```

Comment:
This wasn't technically necessary, but I think it helps code readability in these template files.